datalad_next/shell/tests/test_shell.py

```diff
@@ -13,10 +14,7 @@
     on_windows,
     skip_if,
 )
-from datalad_next.runners import (
-    CommandError,
-    iter_subproc,
-)
+from datalad_next.exceptions import CommandError
```

Comment:
These are two different exceptions. `datalad_next.exceptions.CommandError` is datalad-core's exception, and `datasalad.runners.CommandError` is different (and presently unrelated to that).

Right now that code uses the shim, which raises datalad-core's exception, hence this one is imported.

This will hopefully become simpler when https://github.com/datalad/datalad/pull/7618 (or something like it) makes it into datalad-core.

Reply:
I see, thanks for the explanation. I was not aware of that...